okhttp-tests/src/test/java/com/squareup/okhttp/TestUtil.java

```diff
@@ -15,4 +16,8 @@ private TestUtil() {
     }
     return result;
   }
+
+  public static boolean isEqualCollections(Collection<?> a, Collection<?> b) {
```

Comment:
This isn't well defined. For example, this says that the following lists are equal: [A, B, C], [B, A, A]. I think you want to fix the call site to do set equality, which is well defined.

Perhaps add this helper method:

```
public <T> Set<T> setOf(T... elements) {
  return setOf(Arrays.asList(elements)); 
}
public <T> Set<T> setOf(Iterable<T> elements) {
  return new LinkedHashSet<>(elements); 
}
```

And then you can do this:

```
assertEquals(setOf(keypairACertificate1PinBase64, keypairBCertificate1PinBase64), setOf(matchedPins));
```

This isn't off the top of my head; I've run into this before!